frontend/src/metabase/dashboard/hooks/use-embed-theme.ts

```diff
@@ -29,12 +27,6 @@ export const useEmbedTheme = (
 
   const hasNightModeToggle = theme !== "transparent";
 
-  useEffect(() => {
```

Comment:
This can be removed because from the change above it'd be transformed to
```ts
useEffect(() => {
  setTheme("light")
}, [setTheme]);
```

And we don't need to do this since `light` is already the default value in Redux (`setTheme` will set the value back to this slice of the state below)
https://github.com/metabase/metabase/blob/19cbeacb939ed9d6953b9de804f89ff041b5bdf1/frontend/src/metabase-types/store/mocks/dashboard.ts#L33